pyaedt/desktop.py

```diff
@@ -261,115 +270,186 @@ def _delete_objects():
     for p in sys.path[::-1]:
         if "AnsysEM" in p:
             del sys.path[sys.path.index(p)]
-    gc.collect()
+    # gc.collect()
 
 
 @pyaedt_function_handler()
-def release_desktop(close_projects=True, close_desktop=True):
+def _close_aedt_application(close_desktop, pid, is_grpc_api):
     """Release the AEDT API.
 
     Parameters
     ----------
-    close_projects : bool, optional
-        Whether to close the AEDT projects open in the session. The default is ``True``.
-    close_desktop : bool, optional
-        Whether to close the active AEDT session. The default is ``True``.
+    close_desktop : bool
+        Whether to close the active AEDT session.
+    pid : int
+        PID of the desktop application that is being closed.
```

Comment:
```suggestion
        PID of the desktop app that is being closed.
```
Google developer documentation style guide, which we follow, says to use "app" rather than "application". Instead of starting with "PID", can we say start with "Product ID"?

Reply:
@Alberto-DM We use ID for identifier everywhere in PyAnsys documentation. I'd edited it to use Process ID. The parameter name uses PID, so I don't think we need to define it in the description itself.